packages/SwingSet/src/kernel/vat-admin-hooks.js

```diff
@@ -112,5 +112,20 @@ export function makeVatAdminHooks(tools) {
       terminateVat(vatID, true, reasonCD);
       return harden({ body: stringify(undefined), slots: [] });
     },
+
+    changeOptions(argsCapData) {
+      // marshal([vatID, options]) -> null
+      assert(argsCapData.slots.length === 0);
```

Comment:
Let's add a message of some sort here, this "shouldn't happen" since vat-vat-admin is doing the primary checks, but if something slips through, it'd be nice to see `changeOptions() refuses slots` or something we can grep for. (I've been burned by empty assertions one too many times, they can get hard to track down when stacks are broken for one reason or another).